src/wp-includes/blocks.php

```diff
@@ -1582,6 +1584,133 @@ function make_after_block_visitor( $hooked_blocks, $context, $callback = 'insert
 	};
 }
 
+function apply_block_bindings_to_block( &$parsed_block, $parent_block, $context ) {
```

Comment:
This is pretty much copied from `WP_Block::process_block_bindings`. Ideally, the two functions would share most of their code.